cypress/integration/Tab.spec.js

```diff
@@ -22,7 +22,7 @@ describe('The Tabs component', () => {
       // Setting viewport dimensions to avoid side effects
       cy.viewport(1400, 600);
       cy.get('button')
-        .first()
+        .eq(1)
```

Comment:
Oh I see why the test changed – were clicking on the second item instead of the first